docs/references/general/dat.md

```diff
@@ -0,0 +1,113 @@
+| *frang*                | *km*      | short     | Distance to the first range gate                                                                                                                                                                 |
+| *rsep*                 | *km*      | short     | Range separation                                                                                                                                                                                 |
+| *xcf*                  | *None*    | short     | XCF flag, indicating whether the power, velocity, spectral width and phi0 parameters fitted from the cross-correlation function are available (xcf=1) or not available (xcf=0)                   |
+| *tfreq*                | *kHz*     | short     | Transmitted frequency                                                                                                                                                                            |
+| *mxpwr*                | *None*      | int       | Maximum power                                                                                                                                                                                    |
+| *lvmax*                | *None*    | int       | Maximum noise level allowed                                                                                                                                                                      |
+| usr_resL2              | *None*    | int       | User defined variable                                                                                                                                                                            |
+| cp                     | *None*     | short     | Control program number                                                                                                                                                                           |
+| usr_resS1              | *None*    | short     | User defined variable                                                                                                                                                                            |
+| usr_resS2              | *None*    | short     | User defined variable                                                                                                                                                                            |
+| usr_resS3              | *None*    | short     | User defined variable                                                                                                                                                                            |
+| *combf*                | *None*    | string    | Comment buffer, usually contains the control program name                                                                                                                                        |
```

Comment:
Maybe we can list the user-defined variables together at the end? And does it make sense to write them more compactly, i.e.

```suggestion
| cp                   | *None*    | short     | Control program number 
|                                                                                                                                                              |
| *combf*              | *None*    | string    | Comment buffer, usually contains the control program name                                                                                                                                        |                                                                                                                                                                    |
| usr_resS[1-3]        | *None*    | short     | User defined variables                                                                                                                                                                            |
| usr_resL[1-2]        | *None*    | long      | User defined variables                                                                                                                                                                            |
```

Reply:
I guess, personally, I like the way I have it because of how datdump prints it out. So consistency I guess. But I am good either way. 

Also `usr_resL1` is normally used for the offset.